tools/minikube/scripts/start_pods.sh

```diff
@@ -87,20 +73,11 @@ fi
 rm -rf $tmp_dir
 
 kubectl create configmap \
-    --namespace=catalog \
-    ansible-controller-env \
-    --from-literal=PINAKES_CONTROLLER_URL="$PINAKES_CONTROLLER_URL" \
-    --from-literal=PINAKES_CONTROLLER_TOKEN="$PINAKES_CONTROLLER_TOKEN" \
-    --from-literal=PINAKES_CONTROLLER_VERIFY_SSL="$PINAKES_CONTROLLER_VERIFY_SSL"
+	--namespace=catalog \
```

Comment:
If you run twice the start_pods script, the command fails because the configmap already exists. I suggest overwrite it: 
```kubectl create configmap \
        --namespace=catalog \
        pinakes-env-overrides \
        --from-env-file=minikube_env_vars -o yaml --dry-run=client | kubectl apply -f -
```
Or delete/create it every execution